Popup/Popup.module.less

```diff
@@ -4,16 +4,31 @@
 @import "../styles/skin.less";
 
 .popup {
-	.applySkins({
-		position: fixed;
+	position: fixed;
 		top: 50%;
 		left: 50%;
 		transform: translate(-50%,-50%);
+		// .position(0);
```

Comment:
Do we need this code? It seems that we don't use it.

Reply:
Ok.